src/status_im2/contexts/chat/home/view.cljs

```diff
@@ -4,6 +4,7 @@
             [i18n.i18n :as i18n]
             [react-native.core :as rn]
             [quo2.core :as quo]
+            [quo.react :as react]
```

Comment:
why are are serving react from quo.react.. can we not move this to `src/react` ? 🤔

Reply:
again perhaps beyond the scope of this p.r, but it could be a follow up.

Reply:
I think so @J-Son89, the react wrapper functions found `quo.react` are quite useful, but they are in the wrong place. Of course it's beyond the scope of this PR, but it'd be nice to extract it out.

Reply:
I'm resolving this thread because I changed the code to use signals to keep the app db synchronized with pending contact requests.